zeekclient/types.py

```diff
@@ -582,11 +582,14 @@ def from_config_parser(cls, cfp, _section=None):
                         config.instances.append(Instance(key))
                     else:
                         hostport = val
-                        parts = hostport.split(':', 1)
-                        if len(parts) != 2 or not parts[0] or not parts[1]:
+                        parts = hostport.rpartition(':')
```

Comment:
Stackoverflow suggests urlparse() might be possible to use with quirk of needing a `//` prefix.
```
from urllib.parse import urlparse
parse_result = urlparse("//" + val)  # need // as scheme separator
config.instances.append(Instance(key, parse_result.hostname, parse_result.port))
```

```
>>> from urllib.parse import urlparse
>>> r = urlparse("//[ffee::1]:43242")
>>> r.hostname
'ffee::1'
>>> r.port
43242
```

Reply:
I like the simplicity of that in the code, but I don't like the idea of introducing yet another address format for the zeek users that's only in this one place.

Reply:
I'm unclear what you mean - the idea would be that the pre-pending of "//" is done internally. It's a bit of a quirk to re-use urlparse. That said, `split` and `rpartition` also work, so I'm resolving.